src/patch.jl

```diff
@@ -0,0 +1,138 @@
+struct KwArgs{T}
```

Comment:
`Keywords` would read better

Reply:
I think I didn't end up using this and this struct should be deleted